go/sigid/data/mkdata.iced

```diff
@@ -0,0 +1,76 @@
+{Gets} = require('iced-utils')
+{make_esc} = require 'iced-error'
+
+hash_to_uint32 = (h) -> "0x" + h[0...8]
+hash_to_3bytes = (h) -> ( "0x#{h[i...(i+2)]}" for i in [0...6] by 2)
+hash_to_uint16 = (h) -> "0x" + h[0...4]
+
+# "abcdef1234" => ["abcd", "abcdef", "abcdef12" ]
+prefix_split = (h) -> (h[0...i] for i in [4..8] by 2)
+
+class Runner
+
+  constructor : () ->
+    map = []
+
+  read : ({input}, cb) ->
+    esc = make_esc cb, "read"
+    gets = (new Gets input).run()
+    h = {} # Sigs that start with 'h' are the fixed/new ones
```

Comment:
maybe link to the pr and comment that his has to do with msgpack encoding of different length structures?

Reply:
what does "-" mean in sigids-1.0.16? neither g nor h? what caused those sigs?

Reply:
PGP